src/OrchardCore.Modules/OrchardCore.Admin/PermissionProvider.cs

```diff
@@ -4,13 +4,11 @@
 
 namespace OrchardCore.Admin;
 
-public class Permissions : IPermissionProvider
```

Comment:
This was moved to the abstractions class since I need to access it from other modules like Notifications.